test/ck_tile/gemm_multi_d/test_gemm_multi_d_util.hpp

```diff
@@ -16,41 +16,25 @@
 struct ElementWiseAddAdd
 {
     template <typename E, typename C, typename D0, typename D1>
-    CK_TILE_DEVICE auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
+    __host__ __device__ auto operator()(E& e, const C& c, const D0& d0, const D1& d1) const -> void
```

Comment:
You have to use CK_TILE_HOST_DEVICE since host device can cause redefinition in ck tile